### ISSUE TITLE
Material Design Upgrade for Quran App

### ISSUE THREAD

**Issue by tahabashir1991** [quran/quran_android#653]
As the material design implemented by google in their apps with left swipe menu is better option . they also implemented the same in kitkat too :) it would be good if you further update the material design :)
i have used two colour present in the icon of app Quran Naskh to show how beautiful it will look on proper material design :) hope you will consider it :)
![final poster](https://cloud.githubusercontent.com/assets/18431434/14810716/bb418014-0bad-11e6-9288-ae5a66dfc042.jpg)

Menu with shade
![final shaded image 2_framed](https://cloud.githubusercontent.com/assets/18431434/14810721/c4747f1a-0bad-11e6-9138-51fef5b4cda3.png)

menu without shade
![final shaded image_framed](https://cloud.githubusercontent.com/assets/18431434/14810723/c86b6804-0bad-11e6-866f-4f0742bb6e35.png)

each section UI . e.g in current pic juz UI after opening from left panel
![new juzz style_framed](https://cloud.githubusercontent.com/assets/18431434/14810738/dbcb6f52-0bad-11e6-8995-7ff9ca808126.png)

**Comment by ozbek** [comment]
Commenting just so you don't close yet this one and open a new: **you don't have to close/open to update the content, please simply edit the existing one.**

**Comment by tahabashir1991** [comment]
sorry for the inconvenience .... i am new to github .. next time i will be careful and thankyou

**Comment by ahmedre** [comment]
jazakAllah khairan - i am not sure about this, though i have thought about it before. first off, our existing design is material (and was designed with material in mind, using the specs, etc). secondly, many people write about the fact that the "hamburger menu" is bad for discoverability (very few people can easily find it, etc) - on the index screen, it's not needed.

on the reading screen, we could add something similar to this - but it'd be even worse than a normal hamburger menu (since it'd be two taps under - one to toggle the menu bar, and a second to open the hamburger menu).

we can think about doing this in the future insha'Allah, but am personally not sure. @ozbek, what do you think?

**Comment by tahabashir1991** [comment]
yes you are right but every upcoming app from google in marshmallow have that side menu e.g gmail ,. play store and people are liking and okay with this design so i thought a platform resemblance would be good so i tried to make one :) but the quran logo behind the juz list .... that can be considered even if you don't go for google design :D

**Comment by ozbek** [comment]
Sorry for late response... I don't think we need the sliding panel at the moment. 

From my personal usage perspective, the most used UI items are already in the tabs (Surahs/Juz'/Bookmarks) or actionbars (Search/Translations/Play). Other menus and options are rarely used.

Yes, we could probably add background images and elevation effects.

**Comment by nyitgroup** [comment]
Very nice designs. If the model and data objects were independent of the controls it should be easy.
